html-css/week2/index.html

```diff
@@ -0,0 +1,158 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
```

Comment:
Hi Yuliia, good job :) I'll leave a few comments ;)

Reply:
Hi Gohar. Thank you for your review.